Sources/SwifterSwift/SwiftStdlib/ArrayExtensions.swift

```diff
@@ -33,6 +33,24 @@ public extension Array {
         guard startIndex..<endIndex ~= otherIndex else { return }
         swapAt(index, otherIndex)
     }
+
+    /// SwifterSwift: Sort an array like another array based on a keyPath. If the other array doesn't contain a certain value, it will be sorted last.
```

Comment:
```suggestion
    /// SwifterSwift: Sort an array like another array based on a key path. If the other array doesn't contain a certain value, it will be sorted last.
```